scripts/delayed.js

```diff
@@ -18,11 +18,6 @@ function loadScript(url, type, callback) {
   }
   return script;
 }
-
-window.targetGlobalSettings = {
-  bodyHidingEnabled: false,
```

Comment:
Why did you remove this again? This was causing a white flash after a few seconds when Target kicks in...

Reply:
I have not deleted it.

Reply:
how is this getting deleted